### DISCUSSION TITLE
perl script error message

### DISCUSSION THREAD

**Discussion by laurel-nave-powers** [sr320/course-fish546-2021#76] [Q&A] [answered]
I ran the help flag on a script I want to use to check if it would work/if I needed modules downloaded and it gave me an error message I haven't seen before and that I'm not sure how to interpret. I opened the script in atom and looked at the lines mentioned in the error message but am still not sure. Any ideas or insight would be appreciated! 
Here is the notebook I'm working in with the error message: https://github.com/fish546-2021/Laurel-genes/blob/main/code/03_rmdup.ipynb 
Here is the script: https://github.com/calderatta/ca-exon-capture/blob/master/pipeline_scripts/assemble/rmdup.pl 
Here is what the script looks like when I open it in Atom, which is different than what it looks like on the GitHub it came from: 
<img width="918" alt="Screen Shot 2021-02-12 at 4 37 14 PM" src="https://user-images.githubusercontent.com/76967600/107836087-ae0f9480-6d50-11eb-8d2f-69c45bac4e52.png">

Is this an issue of me downloading the script wrong?

**Comment by kubu4** [comment]
Yes, the script is downloaded incorrectly.

**Comment by sr320** [comment, marked as answer]
I added the script to your repo... and made a recording of how I did it (I cloned your repo and pushed changes)

see https://d.pr/i/Q6PSzI

Here is notebook show how you might run the -h command. Note I do not have dependencies on my computer so will not operate correctly but imagine it would for you..

https://github.com/fish546-2021/Laurel-genes/blob/main/code/99-SR.ipynb
